duckdns/README.md

```diff
@@ -79,8 +78,8 @@ This option allows you to override the auto-detection and specify an
 IPv6 address manually.
 
 If you specify a URL here, contents of the resource it points to will be
-fetched and used as the address. This enables getting the address from
-an external service or from a local `file:/` URL.
+fetched and used as the address. This enables getting the address using
+a service like https://api6.ipify.org/ or https://ipv6.wtfismyip.com/text
```

Comment:
@pvizeli I think adding the services here is not a good change. I specifically left the services out because they do not actually work. ipv6.wtfismyip.com does not have an IPv4 address so it doesn't work at all without IPv6, and api6.ipify.org returns and IPv4 (!) address when fetched without IPv6 support: https://github.com/rdegges/ipify-api/issues/43. So I think the services should not be mentioned here.



In my tests, one of then (I don't remember which one) didn't work at all because the hostname won't resolve because of IPv6, and the other returns an IPv4 address (!) when fetching f